.travis.yml

```diff
@@ -169,7 +164,7 @@ script:
 
         REL_BRANCH=$(echo $(echo "$TRAVIS_TAG" | cut -d'-' -f1 | rev | cut -d'.' -f2- | rev).x$REL_SUFFIX);
 
-        ./buildscripts/git-release "$REPO_ORG/istgt" "$TRAVIS_TAG" "$REL_BRANCH" || travis_terminate 1;
+        ./buildscripts/git-release "$REPO_ORG/libcstor" "$TRAVIS_TAG" "$REL_BRANCH" || travis_terminate 1;
```

Comment:
what is this steps supposed to do? why this change required?

Reply:
This will help to trigger the release on the downstream repo from Travis... Since with this PR, images will be built from libcstor after i.e only after completing the libcstor Travis images & code will be pushed to corresponding image repositories and source repositories... 

Existing system: Since images are pushed from cstor triggering istgt release from cstor is good(With this PR things will change).
 
Release flow(For creating release images):
Existing flow: openebs/linux-utils ---> openebs/libcstor ---> openebs/cstor ---> openebs/istgt ....
Updated flow: openebs/linux-utils ---> openebs/cstor ---> openebs/libcstor ---> openebs/istgt ....

Reply:
@mittachaitu let us keep the old flow, as libcstor will become the main repo later on (we will probably rename it) which will tag all dependant repo.

Reply:
Ok so you mean cstor and istgt will be triggered from libcstor itself(after renaming it).... that makes sense.

Then I no need to raise a PR in openebs/linux-utils cc: @kmova

Reply:
I see, we need base image first.

Reply:
We are building a base image from libcstor itself https://github.com/openebs/libcstor/blob/35c939ccb7473e2674a3fe9877c33cd74115740a/build_image.sh#L120. So can I change it to(Since base and main images is built from libcstor)
```
                                                   --------> openebs/cstor
openebs/linux-utils ---> openebs/libcstor  ----->  |
                                                   --------> openebs/istgt ----> other dependent repos
```

Reply:
@mittachaitu I see a problem. The image should be pushed at last. And since we are now pushing from libcstor, that should be done after build is successful on cstor. So cstor should be tagged first. 

The other thing is, we any way build everything from libcstor, so tagging cstor does not server any purpose. just that we have marked the code for a perticular tag.

let us go ahead with the way you did, later on we will decide how can we do that.

Reply:
Can I remove the triggering downstream repo from cstor?

Reply:
> let us go ahead with the way you did, later on we will decide how can we do that.

Ok

Reply:
yeah.. triggering builds from cstor can be removed.